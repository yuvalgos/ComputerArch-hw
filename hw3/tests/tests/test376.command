./dflow_calc tests/test376.dat tests/test376.in p77 p184 p133 d96 d123 p119 d37 p133 p73 p78 p169 d142 p21 d36 d198 d5 d173 d135 p17 p69 d8 p180 p183 p34 p149 p145 d27 d67 p195 p120 d160 d76