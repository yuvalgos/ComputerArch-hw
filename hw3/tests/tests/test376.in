6 1 2 1
8 19 20 20
1 0 5 21
2 11 21 24
4 14 17 5
0 9 16 12
1 6 13 29
0 19 1 3
7 28 10 21
6 5 1 12
5 22 10 4
2 15 7 30
0 16 25 10
7 16 30 3
5 30 6 15
7 21 25 19
7 7 25 9
6 30 27 2
8 19 16 16
5 22 21 16
5 17 11 2
2 18 3 4
6 15 29 0
5 3 28 11
2 17 4 19
7 0 31 30
6 2 9 8
4 19 8 18
3 17 24 24
0 7 13 17
2 5 7 4
8 6 9 2
7 27 16 7
6 27 23 16
3 14 2 1
8 9 25 12
1 14 20 25
0 15 27 1
1 5 19 23
4 29 9 26
3 26 22 6
7 3 18 13
8 21 16 26
3 29 7 27
5 27 5 24
6 0 29 19
8 26 15 6
1 30 13 18
0 10 17 8
5 3 1 23
3 20 6 2
0 25 21 23
1 15 7 0
7 31 4 5
0 18 13 1
6 26 13 21
3 16 28 26
2 17 12 2
4 31 21 26
2 23 30 6
6 2 4 28
0 23 24 22
4 4 26 1
4 1 3 2
4 30 11 13
7 30 5 17
4 30 13 9
2 24 19 19
8 23 15 15
5 8 21 22
2 3 3 5
4 25 30 3
5 17 3 7
7 15 13 25
4 21 23 5
8 11 13 18
0 20 31 19
2 25 11 12
4 3 4 2
7 10 14 3
0 5 19 30
4 0 18 26
5 30 27 7
7 29 10 6
5 14 13 18
6 1 17 13
1 9 19 31
5 12 28 13
5 1 22 11
4 5 23 30
0 17 18 9
6 13 1 7
6 31 24 26
0 11 17 10
7 24 29 23
4 3 5 12
7 15 12 17
3 29 14 17
3 31 16 10
6 0 17 5
6 3 21 31
3 26 16 22
4 27 29 0
4 0 24 20
1 0 22 14
3 17 15 11
5 0 14 2
2 24 17 3
8 26 9 16
0 2 1 1
1 25 0 20
2 31 10 18
5 29 5 8
1 31 7 23
7 16 13 0
3 3 5 16
1 5 11 4
1 20 29 24
7 17 3 8
6 17 4 10
6 8 8 25
6 26 18 24
7 30 15 5
8 8 11 31
1 10 15 7
0 1 8 29
0 21 3 16
6 1 18 11
2 16 25 0
6 13 12 21
7 18 3 2
4 23 13 30
1 5 19 23
2 28 2 6
5 5 26 19
3 17 9 30
6 15 5 29
6 20 28 12
2 27 27 31
6 9 3 7
0 6 19 26
0 7 22 5
5 18 27 5
0 7 10 1
3 25 18 6
5 31 18 10
7 3 12 28
8 8 20 2
1 29 13 16
0 12 19 14
3 3 4 20
7 5 23 31
6 18 21 27
4 2 9 1
4 11 19 27
3 24 4 14
1 24 20 11
2 6 3 3
1 5 22 15
4 14 13 22
3 24 9 16
4 20 30 8
1 16 9 13
5 8 22 12
8 2 16 30
2 15 24 8
7 29 0 13
5 12 1 21
8 24 25 13
3 22 6 30
7 9 11 31
5 8 27 18
4 23 19 21
4 20 12 13
1 18 31 19
5 11 27 3
3 29 18 26
3 3 5 17
2 18 23 22
1 24 1 21
0 21 4 28
1 30 31 0
4 31 3 3
4 21 29 7
5 2 31 29
1 20 28 4
0 19 25 25
1 30 10 20
6 28 0 10
6 7 9 13
7 24 0 22
4 27 27 18
7 28 6 3
8 5 12 11
2 27 5 29
8 10 20 7
2 26 20 26
1 7 9 4
8 0 14 6
5 0 11 27
