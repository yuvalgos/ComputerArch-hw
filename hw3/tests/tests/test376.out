Reading the opcodes latency info from tests/test376.dat ... Got latency for 9 opcodes
Reading the program file tests/test376.in ... Found 200 instructions
getProgDepth()==283
getDepDepth(77)==42
getDepDepth(184)==236
getDepDepth(133)==66
getInstDeps(96)=={87,90}
getInstDeps(123)=={93,113}
getDepDepth(119)==139
getInstDeps(37)=={33,0}
getDepDepth(133)==66
getDepDepth(73)==46
getDepDepth(78)==109
getDepDepth(169)==163
getInstDeps(142)=={138,134}
getDepDepth(21)==0
getInstDeps(36)=={-1,-1}
getInstDeps(198)=={159,157}
getInstDeps(5)=={-1,-1}
getInstDeps(173)=={167,129}
getInstDeps(135)=={86,122}
getDepDepth(17)==0
getDepDepth(69)==81
getInstDeps(8)=={-1,-1}
getDepDepth(180)==109
getDepDepth(183)==234
getDepDepth(34)==19
getDepDepth(149)==32
getDepDepth(145)==211
getInstDeps(27)=={-1,21}
getInstDeps(67)=={27,27}
getDepDepth(195)==260
getDepDepth(120)==114
getInstDeps(160)=={139,128}
getInstDeps(76)=={58,27}
